pyensembl/database.py

```diff
@@ -74,7 +74,7 @@ def _all_possible_indices(self, column_names):
         Create list of tuples containing all possible index groups
         we might want to create over tables in this database.
 
-        If a release is missing some column we want to index on,
+        If a genomic database is missing some column we want to index on,
```

Comment:
I tried, but didn't quite succeed, to use consistent verbiage throughout: mostly because I was conflicted about it!

How do I refer to the collection of stuff that includes GTF and FASTA? Annotation seems good for just the GTF part?